### ISSUE TITLE
additional_disks acts as required parameter

### ISSUE THREAD

**Issue by mahsoud** [test-kitchen/kitchen-hyperv#46]
If additional_disks parameter is missing from kitchen file and error is thrown on create step:
```
I, [2017-03-28T16:05:37.892758 #5752]  INFO -- InstanceName: -----> Creating <InstanceName>...
I, [2017-03-28T16:05:39.267837 #5752]  INFO -- InstanceName: Creating differencing disk for InstanceName.
I, [2017-03-28T16:05:41.642886 #5752]  INFO -- InstanceName: Created differencing disk for InstanceName.
I, [2017-03-28T16:05:41.642886 #5752]  INFO -- InstanceName: Checking for existing virtual machine.
I, [2017-03-28T16:05:41.642886 #5752]  INFO -- InstanceName: Creating virtual machine for InstanceName.
E, [2017-03-28T16:05:41.642886 #5752] ERROR -- InstanceName: Create failed on instance <InstanceName>.
E, [2017-03-28T16:05:41.642886 #5752] ERROR -- InstanceName: ------Exception-------
E, [2017-03-28T16:05:41.642886 #5752] ERROR -- InstanceName: Class: NoMethodError
E, [2017-03-28T16:05:41.642886 #5752] ERROR -- InstanceName: Message: undefined method `join' for nil:NilClass
Did you mean?  JSON
E, [2017-03-28T16:05:41.642886 #5752] ERROR -- InstanceName: ----------------------
E, [2017-03-28T16:05:41.642886 #5752] ERROR -- InstanceName: ------Backtrace-------
E, [2017-03-28T16:05:41.642886 #5752] ERROR -- InstanceName: C:/Users/UserName/AppData/Local/chefdk/gem/ruby/2.3.0/gems/kitchen-hyperv-0.4.0/lib/kitchen/driver/powershell.rb:126:in `new_vm_ps'
E, [2017-03-28T16:05:41.642886 #5752] ERROR -- InstanceName: C:/Users/UserName/AppData/Local/chefdk/gem/ruby/2.3.0/gems/kitchen-hyperv-0.4.0/lib/kitchen/driver/hyperv.rb:151:in `create_virtual_machine'
E, [2017-03-28T16:05:41.642886 #5752] ERROR -- InstanceName: C:/Users/UserName/AppData/Local/chefdk/gem/ruby/2.3.0/gems/kitchen-hyperv-0.4.0/lib/kitchen/driver/hyperv.rb:68:in `create'
E, [2017-03-28T16:05:41.642886 #5752] ERROR -- InstanceName: C:/opscode/chefdk/embedded/lib/ruby/gems/2.3.0/gems/test-kitchen-1.13.2/lib/kitchen/instance.rb:456:in `public_send'


```

Work around is to add this line in kitchen file:
additional_disks: {}

**Comment by mahsoud** [comment]
poke @smurawski  :)

**Comment by usepowershell** [comment]
closed via #48
